austrakka/components/project/funcs.py

```diff
@@ -30,6 +36,49 @@ def add_project(
         }
     )
 
+@logger_wraps()
+def update_project(
+        project_abbreviation: str,
+        abbrev: str,
+        name: str,
+        description: str,
+        is_active: bool,
+        org: str,
+        dashboard_name: str):
+    project = get_project_by_abbrev(project_abbreviation)
+    
+    # ProjectDTO fields which should go in ProjectPutDTO
+    put_project = {k: project[k] for k in [
+        'abbreviation',
+        'name',
+        'description',
+        'isActive',
+        'requestingOrg',
+        'dashboardName'
+    ]}
+    if project['requestingOrg'] is None:
+        put_project['requestingOrg'] = {'abbreviation': None}
+    
+    if abbrev is not None:
+        logger.warning(f"Updating project abbreviation from {project['abbreviation']} to {abbrev}")
+        put_project['abbreviation'] = abbrev
+    if name is not None:
+        put_project['name'] = name
+    if description is not None:
+        put_project['description'] = description
+    if is_active is not None:
+        put_project['isActive'] = is_active
+    if org is not None:
+        put_project['requestingOrg'] = {
+            "abbreviation": org
+        }
+    if dashboard_name is not None:
+        put_project['dashboardName'] = dashboard_name
+    
+    return api_put(
+        path=path.join(PROJECT_PATH, str(project_abbreviation)),
```

Comment:
Let's change this to f'{PROJECT_PATH}/{project_abbreviation}' for windows users

Reply:
You're right, join should not be used for URL paths